tool/scanslib/utility.py

```diff
@@ -1,7 +1,23 @@
+import datetime
 import re
 
 
 class Utility(object):
+    @classmethod
+    def iso8601_yesterday(cls):
```

Comment:
can pull this function out and just use iso8601_arbitrary_days_ago and pass in 1. 
kind of redundant to use iso8601_yesterday.

Reply:
Done.